File: test/corpus/basic/03-go-api-compatibility/case.ts
// biome-ignore lint/suspicious/noShadowRestrictedNames: Go API compatibility requires this name
import { ANSIColor, RGBColor, String } from '../../../../src/go-style.js';

// Test global convenience functions - using Go-style API exactly
console.log(String('Global Red').Foreground(RGBColor('#FF0000')).String());
console.log(String('Global Blue').Bold().Foreground(RGBColor('#0000FF')).String());

// Test ANSI colors
console.log(String('ANSI Red').Foreground(ANSIColor(1)).String());
console.log(String('ANSI Blue').Foreground(ANSIColor(4)).String());

File: test/corpus/basic/03-go-api-compatibility/case.go
package main

import (
	"fmt"

	"github.com/muesli/termenv"
)

func main() {
	// Test global convenience functions
	fmt.Println(termenv.String("Global Red").Foreground(termenv.RGBColor("#FF0000")))
	fmt.Println(termenv.String("Global Blue").Bold().Foreground(termenv.RGBColor("#0000FF")))
	
	// Test ANSI colors
	fmt.Println(termenv.String("ANSI Red").Foreground(termenv.ANSIColor(1)))
	fmt.Println(termenv.String("ANSI Blue").Foreground(termenv.ANSIColor(4)))
}